cugraph/traversal/BFS.ipynb

```diff
@@ -259,9 +259,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Vertex: 22 was reached from vertex 33 and distance to start is 3\n",
-      "Vertex: 33 was reached from vertex 9 and distance to start is 2\n",
-      "Vertex: 9 was reached from vertex 3 and distance to start is 2\n"
+      "Vertex: 22 was reached from vertex 1 and distance to start is 1\n"
```

Comment:
I think renumbering should be set to `True` in this notebook as the input dataset is 1-based. Otherwise, cugraph will creates vertex 0 which I doubt is intended.

Reply:
@afender `renumbering = false` was @BradReesWork 's original fix... PR #310   Can you push a quick commit wit h a proposed better fix, test it, and let me know so that i can test it?